examples/inc_cylinder_2d/inc_cylinder_2d.ini

```diff
@@ -0,0 +1,80 @@
+[backend]
+precision = double
+rank-allocator = linear
+
+[backend-openmp]
+;cblas = Enter path to local BLAS library for OpenMP backend
+
+[constants]
+nu = 0.008
+Uin = 1.0
+Vin = 0.0
+Pc = 1.0
+ac-zeta = 6.0
+
+[solver]
+system = ac-navier-stokes
+order = 3
+
+[solver-time-integrator]
+formulation = dual
+scheme = bdf2
+pseudo-scheme = rk4
+controller = none
+tstart = 0.0
+tend = 60.0
+dt = 0.0044
+pseudo-dt = 0.0022
+pseudo-niters-max = 25
+pseudo-niters-min = 15
+pseudo-resid-norm = l2
+pseudo-resid-tol = 1e-5
+
+[solver-interfaces]
+riemann-solver = rusanov
+ldg-beta = 0.5
+ldg-tau = 0.1
+
+[solver-interfaces-line]
+flux-pts = gauss-legendre
+
+[solver-elements-tri]
+soln-pts = williams-shunn
+
+[solver-elements-quad]
+soln-pts = gauss-legendre
+
+[soln-plugin-nancheck]
+nsteps = 50
+
+[soln-plugin-pseudostats]
+flushsteps = 20
+file = residual.csv
+header = true
+
+[soln-plugin-writer]
+dt-out = 5.0
+basedir = .
+basename = cylinder_2d_{t:.2f}
+
+[solver-source-terms]
```

Comment:
I'm sure there will be questions about damping the strong initial wave in the transient phase. Then we can just refer to this case. One could pseudo march it to death, but being high-order and using rk4 as a pseudo-stepper, it would take a long time to dissipate it!